UPGRADING.rst

```diff
@@ -10,6 +10,13 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#2025 Register indexer service accounts with SAM
+================================================
+
+Service accounts are now registered with SAM upon deployment.
```

Comment:
Why plural? Isn't there just one SA?

Reply:
Per deployment, yes. I was thinking in terms of the entire team. I'll change to singular since it seems less prone to confusion.